Temporary_PHPCompatibility_Test.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+// Should tell me return void is not available in PHP 5.6
+class Foo {
+	public function do_nothing(): void {
+
```

Comment:
* :no_entry_sign: Missing doc comment for function do_nothing()
* :no_entry_sign: void return type is not present in PHP version 7.0 or earlier

<sub>hash: 3d3b3964061b197aba8324456a398b48</sub>